packages/server/src/routes/business/postBusiness.ts

```
@@ -0,0 +1,122 @@
+// an spec for /business/
+
+import { createRoute, z } from "@hono/zod-openapi";
+import { CreateBusinessSchema, GetBusinessSchema } from "../../drizzle/schema";
+import { AppRouteHandler } from "../../base/type";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)